pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -46,6 +46,11 @@ in stdenv.mkDerivation {
 
     patchShebangs .
     substituteInPlace data/installed-tests/fwupdmgr.test.in --subst-var-by installedtestsdir "$installedTests/share/installed-tests/fwupd"
+
+    # /etc/daemon.conf
+    substituteInPlace meson.build --replace \
```

Comment:
Could you use a patch file instead? It will be more clear when the pattern no longer matches.

Reply:
I also just realized that there's a patch touching the config file already: https://github.com/NixOS/nixpkgs/blob/4207f01f27fb6f3ba22c4720d529812601ad87f1/pkgs/os-specific/linux/firmware/fwupd/fix-paths.patch#L12-L17

I'll investigate further later